### ISSUE TITLE
Use ￼￼🚫 instead of grey dashes to keep people from driving over collapsed bridges

### ISSUE THREAD

**Issue by jidanni** [gravitystorm/openstreetmap-carto#3916]
I don't think grey dashes will keep people from attempting to drive over collapsed bridges.

I think as a safety precaution there should be as many red
- 🚫, or
- ⛔

as the number of grey dashes.

http://focustaiwan.tw/news/asoc/201910010003.aspx
https://www.openstreetmap.org/way/145628516#map=17/24.58501/121.87045

**Comment by jidanni** [comment]
Here the grey dashes might be seen as just one more type of lane divider,
![12224-0](https://user-images.githubusercontent.com/1230959/65997121-5eb62480-e4cb-11e9-9ecc-b8e9a1da55e7.jpg)

We're talking safety issues here.

**Comment by jeisenbe** [comment]
I read about this bridge collapse. Since the bridge no longer exists, it's incorrect to simply tag it as `access=no` which is an access restriction for real features that still are present. A lifecycle prefix like destroyed:highway=* would be more appropriate tagging for this feature. In that case it would no longer render as a highway. (I see that this is already the tagging, but for some reason it is also tagged as highway=unclassified in addition to destroyed:highway=unclassified -https://www.openstreetmap.org/way/145628516/history) 

A bridge that was closed to all traffic would render as above. There are been some suggestions about changing the access rendering in #3645 which is partially a duplicate of this issue. Also see #3724 about z16 with `highway=service` + `access=no`. 

The specific suggestion to use 🚫 or ⛔ icons for `access=no` is not feasible for this map style at all zoom levels, however: adding one icon at the center would still lead to a very busy rendering for areas like military bases with many `access=no` roads, especially where highway ways are split at intersections, which would lead to duplicate icons on each way segment.

So I believe this issue can be closed as a duplicate of #3645.

**Comment by jidanni** [comment]
Indeed as there are [https://wiki.openstreetmap.org/wiki/Talk:Key:bridge#Add_.22what_to_do_in_case_of_disaster.22_section](no SOP yet) in such cases, each time there are such worldwide news making bridge collapses many editors pitch in trying to do (their own version of) the right thing.

**Comment by jidanni** [comment]
Yes ripping the bridge off the map will make it clear to drivers that they are not going to get across the river. Good!
However mappers might worry:
- "Well the bridge is actually still there, although lying *in* the river. I dare not remove it completely."
- "The Minister said they will get to work rebuilding the bridge as soon as the human rescue operation is finished. So the bridge is merely temporarily closed, one might say. I can't just rip it out like it never existed."

**Comment by polarbearing** [comment]
@jidanni - your argumentation is hard to follow. I have no idea what you mean by [SOP](https://en.wikipedia.org/wiki/SOP), too many meanings. Please remember that the carto style can only represent a subsection from the OSM data, and is not a traffic sign map. 
There are several lifecycle tags to use for the stages of replanning and reconstruction, and as soon as work has started you can use the construction tag which will be rendered here.

**Comment by jidanni** [comment]
https://en.wikipedia.org/wiki/Standard_operating_procedure

All I know is with people blindly following electronic maps into
mud holes and off cliffs these days, it should be extra clear that they
should not attempt driving on it, even when viewed on a dimly lit
screen. Those grey dashes just don't pass the test.
(Sure, if the bridge is no longer present on the map, the problem goes away.)

**Comment by kocio-pl** [comment]
But this is rendering style, not a routing service. I don't believe anybody would drive looking at this particular map, not hearing their GPS assistant.

**Comment by jidanni** [comment]
(Off topic: All I know is I use these maps in real life all the time,
loaded into
https://play.google.com/store/apps/details?id=net.psyberia.offlinemaps
But the offline nature means I'll often be working with cached tiles...
so can't blame the map when I end up in the water.

Hmmm, the cycle and transport layers aren't expected to be updated fast
either, and who knows what
https://play.google.com/store/apps/details?id=com.mapswithme.maps.pro
uses instead of those dashes...)
